rskj-core/src/main/java/co/rsk/peg/bitcoin/FlyoverRedeemScriptBuilderImpl.java

```diff
@@ -22,6 +25,7 @@ public static FlyoverRedeemScriptBuilderImpl builder() {
     @Override
     public Script of(Keccak256 flyoverDerivationHash, Script redeemScript) {
         validateFlyoverDerivationHash(flyoverDerivationHash);
+        validateRedeemScript(redeemScript);
```

Comment:
```suggestion
        validateInternalRedeemScript(redeemScript);
```
Just to clear, what do you think?

Reply:
Updated